cmd/core-command/res/docker/configuration.toml

```diff
@@ -49,6 +49,6 @@ Path = '/v1/secrets/edgex/core-command/'
 Protocol = 'http'
 RootCaCertPath = '/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem'
 ServerName = 'edgex-vault'
+TokenFile = '/vault/config/assets/resp-init.json'
```

Comment:
As per my conversation with @tingyuz, the Vault token file will be available to the core-service Docker containers at this location. The PR related to providing this file is https://github.com/edgexfoundry/developer-scripts/pull/139